### DISCUSSION TITLE
Porting to a Mac?

### DISCUSSION THREAD

**Discussion by seishonagon** [tallguyjenks/Obsidian-For-Business#11] [Ideas]
As far as I can tell, Outlook for Mac does not support VBA, so I'd be interested to see if it's possible to port the VBA macros to AppleScript. My AppleScript fu is very low, though, so I'd love help from anybody else out there who's interested.

**Comment by classicrob** [comment]
I'm having the same issue. Curious your thoughts on this.

**Comment by shawngarlandgit** [comment]
Would love the same!
